helpers/pretests_check.rb

```diff
@@ -4,20 +4,31 @@
 # class with methods for check working system before run tests
 class PretestsCheck
   def self.pretests_check
+    tmp_dir_check = FileHelper.dir_check_or_create 'files_tmp'
```

Comment:
I don't think we should store value in variable, since this method in current variant ALWAYS `return true`